packages/graphql-auth-transformer/src/AuthRule.ts

```diff
@@ -2,7 +2,12 @@ export type AuthStrategy = 'owner' | 'groups' | 'public' | 'private';
 export type AuthProvider = 'apiKey' | 'iam' | 'oidc' | 'userPools';
 export type ModelQuery = 'get' | 'list';
 export type ModelMutation = 'create' | 'update' | 'delete';
-export type ModelOperation = 'create' | 'update' | 'delete' | 'read';
+export enum ModelOperation {
```

Comment:
If we are giving the enums string values is there a pro over using union types?

Reply:
The pro is that you can enumerate the values at runtime using ModelOperation.ENUM (as I do when constructing the rule operation filters). When using string unions, the union is gone at runtime so you just have to use raw strings which is not typesafe and difficult to refactor